js/theme/src/global.css

```diff
@@ -1,9 +1,9 @@
 /*
  Custom element styles
  */
-gradio-lite {
+/* gradio-lite {
```

Comment:
I remember I added this because it was necessary for the embedded app to be styled correctly atm.
Probably it's behaving the same as the following style which is for the normal Gradio.
https://github.com/gradio-app/gradio/blob/aba44707af20b78a7b87f5f7bd3f189d971afc71/js/app/index.html#L95

If Lite works well without it now, this can be removed.

Reply:
Restored it